packages/management/tkr-service/bundle/config/upstream/certificates.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  labels:
+  name: tkr-resolver-cluster-webhook-selfsigned-issuer
+  namespace: tanzu-system
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+  name: tkr-resolver-cluster-webhook-serving-cert
+  namespace: tanzu-system
```

Comment:
Same as above here and in the next few files. 
> Should this be in the newly created tkr-system namespace?

Reply:
Same as above. Will update the namespace to `tkr-system`

Reply:
Done